index.html

```diff
@@ -29,7 +27,108 @@ <h1>I'm Egbe Bless Ashuakie!</h1>
                 <li><a href=""><img src="images/rabbit.png" alt=""></a></li>
             </ul>
         </div>
-        
+        <div class="cards">
```

Comment:
- [ ] Kindly ensure your work section is wrapped using the `<section>` tag, currently you are using a `div`.